modin/utils.py

```diff
@@ -527,6 +527,42 @@ def try_cast_to_pandas(obj: Any, squeeze: bool = False) -> Any:
     return obj
 
 
+def repartition(df, axis: Optional[int] = None):  # type:ignore
+    """
+    Repartitioning Modin objects to get ideal partitions inside.
+
+    Allows to improve performance where the query compiler cannot yet.
+
+    Parameters
+    ----------
+    df : DataFrame or Series
+    axis : int, optional
+
+    Returns
+    -------
+    DataFrame or Series
+        The repartitioned dataframe or series, depending on the original type.
+    """
+    if axis not in (0, 1, None):
+        raise NotImplementedError
```

Comment:
can you also add a check that the call of `repartition()` with the selected execution makes sense?

Reply:
Do you mean if it's HDK then just return the DataFrame or Series unchanged?

Reply:
yes

Reply:
ok, added